src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -1758,6 +1757,38 @@ public function orderBefore($uuid, $beforeUuid, $userId, $webspaceKey, $language
     }
 
     /**
+     * {@inheritDoc}
+     */
+    public function orderAt($uuid, $position, $userId, $webspaceKey, $languageCode)
```

Comment:
should we provide an order event (instead of order before and order at)? otherwise we should add here an event order at ... /cc @danrot